src/app/action/action.html

```diff
@@ -40,8 +39,9 @@
       <li ng-repeat="item in action.items">
         <article class="panel panel-default">
           <div class="panel-heading fixed-panel-head">
-            {{ item.name }}
             <span type="button" class="btn btn-default btn-xs glyphicon glyphicon-thumbs-up pull-right"></span>
```

Comment:
put icons as separate tag like `<button><i class="glyphicon glyphicon-thumbs-up"></i></button>
btw why do we need like for actions?
also never use `<span>` for button